### DISCUSSION TITLE
Usage of other git platforms?

### DISCUSSION THREAD

**Discussion by rstub** [r-universe-org/help#463] [Q&A] [answered]
I am considering using codeberg.org for some future packages. From what I can tell, that should work out of the box. Only for supporting releases it would be necessary to add something like the GitLab specific code from: https://github.com/r-universe-org/sync/commit/5ee56aeb75de6effd3f38fe23597bbe121c32cfb
Correct?

**Comment by maelle** [comment, marked as answer]
Yes it should work! https://docs.r-universe.dev/publish/set-up.html#how-much-github-is-compulsory
